style.css

```
@@ -21,16 +21,23 @@ body {
 	box-sizing: border-box;
 }
 
-
 /*Header*/
 
 .header {
+	position: fixed;
+	width: 100%;
+	background-color: #fff;
+	padding: 8px 0 8px;
+}
+
+.header_container {
 	display: flex;
 	align-items: center;
 	justify-content: space-between;
-	max-width: 1140px;
 	margin: 0 auto;
-}
+	max-width: 1150px;
+	padding: 0 10px 0;
```

Aqui tb dá pra simplificar:
```suggestion
	padding: 0 10px;
```